### ISSUE TITLE
load gif formatted with PREFER_RGB_565,  black background appears under the transparent gif

### ISSUE THREAD

**Issue by z360394624** [bumptech/glide#3023]
when i load a gif from network. If I format with PREFER_RGB_565, there is a black background appears under the transparent gif.

```java
 GlideApp.with(imageView)
                    .load("http://www.lucius.tech/cache/comm_loading_header.gif")
                    .format(DecodeFormat.PREFER_RGB_565)
                    .into(imageView);
```
It will be displayed correctly formatted with PREFER_ARGB_8888

```java
 GlideApp.with(imageView)
                    .load("http://www.lucius.tech/cache/comm_loading_header.gif")
                    .format(DecodeFormat.PREFER_ARGB_8888)
                    .into(imageView);
```

**Comment by sjudd** [comment]
Hi @z360394624,

It looks like you might not have included some important information when you filed this issue. To make sure we have all the information we need to help with your issue. Please take a look at the [Issue Template](https://github.com/bumptech/glide/blob/master/ISSUE_TEMPLATE.md) and make sure you've filled out as many sections as possible.

In particular you **must** include:

* Glide Version
* Glide load line
* Issue details.

I'm marking this issue as requiring more information. Please understand that if you do not provide that information or update the issue within the next 7 days this issue will be automatically closed so that it doesn''t clutter the bug tracker.

Best regards,

~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by wangrunxiang** [comment]
I have the same problem @sjudd

**Comment by sjudd** [comment]
If someone can reproduce this, can you attach an example GIF to the issue? The link in the original seems to no longer be valid.

**Comment by sjudd** [comment]
This is fixed internally by cl/236478315
